examples/nlp_seq/README.md

```diff
@@ -21,7 +21,7 @@ The model should run with other configurations and hardware, but explicitly test
 
 | Hardware |  Batch size  | Learning rate | Training time | Accuracy  | TensorBoard.dev |
 |:---:|:---:|:---:|:---:|:---:|:---:|
-| Nvidia V100 (16GB) | 64  |  0.05 | 5h 15m | 72.20% | [2020-03-22](https://tensorboard.dev/experiment/YkUAdwYaQ9OtYl2IVe3MvA/) |
+| Nvidia Titan V (12GB) | 64  |  0.05 | 1:36h | 68.6% | [2022-04-29](https://tensorboard.dev/experiment/F5ULHlyzQlieVJn5PG8mRQ/) |
```

Comment:
hm... the linked tensorboard shows `xpos_eval=0.6928` after `5h58m21s` ...

how come this decreased so much from the previous 72.20% ?

Reply:
I do not know what cause this. I have read the code again and as it is only an education example, I would suggest to go ahead. We might investigate later again.